### ISSUE TITLE
Mac Monterey windows not being reorganized when being shown after being hidden

### ISSUE THREAD

**Issue by brandonkboswell** [ianyh/Amethyst#1255]
Here is a video demonstrating the bug and how to reproduce: https://www.loom.com/share/7e8d56503db6456c8b62419528061df2

**Describe the bug**
Mac Monterey windows not being reorganized when being shown after being hidden

**Applications:**
I seem to experience this only if Chrome is one of the windows, but that may be a red herring.

**To Reproduce**
Steps to reproduce the behavior: Have a two window layout with chrome on the left, hide the window on the right, close the window on the left, reshow the window on the right (in my case with the hammerspoon launchOrFocus command)

**Expected behavior**
I would expect the window to take up the whole screen as it is the only window, but instead it stays taking up half of the screen on the right. 

**Screenshots**
See the loom above.

**Versions:**
 - macOS: Monterey 12.5 Beta
 - Amethyst: 0.15.6 (93)
 
**Debug Info**
```
$ /Applications/Amethyst.app/Contents/MacOS/Amethyst --debug-info
```
Version: 0.15.6 (93)

OS version: Version 12.5 (Build 21G5037d)

Screens:
	(0.0, 0.0, 1920.0, 1080.0) [(0.0, 0.0, 1920.0, 1080.0)]
	(-1440.0, -1050.0, 1440.0, 2560.0) [(-1440.0, -430.0, 1440.0, 2560.0)]

Configuration:
layouts: (
    tall,
    fullscreen
)
ignore-menu-bar: 1
mod1: (
    option,
    shift,
    control,
    command
)
mouse-follows-focus: 1
window-minimum-height: 0
screen-padding-top: 0
window-minimum-width: 0
window-margins: 0
screen-padding-left: 0
mouse-swaps-windows: 1
use-canary-build: 0
new-windows-to-main: 0
debug-layout-info: 1
follow-space-thrown-windows: 1
window-margin-size: 0
floating: (
        {
        id = "com.loom.desktop";
        "window-titles" =         (
        );
    },
        {
        id = "com.skitch.skitch";
        "window-titles" =         (
        );
    },
        {
        id = "com.microsoft.autoupdate2";
        "window-titles" =         (
        );
    },
        {
        id = "com.adobe.Photoshop";
        "window-titles" =         (
        );
    },
        {
        id = "net.matthewpalmer.Vanilla";
        "window-titles" =         (
        );
    },
        {
        id = "com.googlecode.iterm2";
        "window-titles" =         (
        );
    },
        {
        id = "com.akiflow.akiflow";
        "window-titles" =         (
            "",
            "Akiflow - Cli",
            "Akiflow - Notifications"
        );
    },
        {
        id = "com.obsproject.obs-studio";
        "window-titles" =         (
        );
    },
        {
        id = "com.lukebrody.Teleprompter";
        "window-titles" =         (
        );
    }
)
restore-layouts-on-launch: 0
screen-padding-bottom: 0
enables-layout-hud: 0
focus-follows-mouse: 0
screen-padding-right: 0
floating-is-blacklist: 1
enables-layout-hud-on-space-change: 0
mouse-resizes-windows: 1
window-resize-step: 8
mod2: (
    option,
    command,
    control
)
float-small-windows: 1

**Additional context**

**Comment by simoleone** [comment]
I get similar behavior and it always seems to involve Chrome. No Chrome no problem. A related behavior I've seen in the past week or so is that any window resize or shuffle seems to "animate" but really slowly. Closing and reopening Chrome fixes it for a little while and then it always comes back... eventually.

**Comment by brandonkboswell** [comment]
This has been my experience exactly. It's nice to hear from someone else that it feels like the same source and is not something unique to my setup.

**Comment by brandonkboswell** [comment]
@simoleone just for trying to track down the scenario where this is occurring. I have a 2 monitor setup at my desk. I don't think I have seen this when on the go with just one monitor. Are you using multiple monitors? Have you experienced this issue with just one monitor?

**Comment by simoleone** [comment]
I usually use a 2 monitor setup. Haven't noticed it on just the one monitor but I usually full-screen all my apps when I'm on one monitor so it might be happening but not as easy to notice.

**Comment by ianyh** [comment]
I think this is addressed in #1276
